helix-term/src/commands.rs

```diff
@@ -360,6 +360,7 @@ impl MappableCommand {
         jump_view_left, "Jump to the split to the left",
         jump_view_up, "Jump to the split above",
         jump_view_down, "Jump to the split below",
+        transpose_view, "Transpose splits",
```

Comment:
This is a great feature! Thank you for adding it

I must say when I read the words "transpose view" I assumed it would swap the contents of two windows, not swap from a horizontal split to a vertical split. However, reading the description "transpose splits" made it immediately clear.